Remove authnz (#408)
repository-service-tuf/repository-service-tuf-cli: docs/source/devel/design.rst

The ``repository-service-tuf``, in the container perspective, is a command line tool that
interacts to the ``repository-service-tuf-api``.

> ``repository-service-tuf`` reads settings configuration in the file

Reviewer: ```suggestion
``repository-service-tuf`` reads the settings configuration from path
```

Reviewer: Nit: the above comment.
